XpressAI/xircuits
# ✨ Add `Any` Argument general component

# Description

This PR adds the `any` type argument to the general component list. This is particularly important for workflows that would like to accept arbitrary data to workflow components.

## Pull Request Type


## Type of Change


# Tests

1. Drag in the Argument `any` component. Confirm that you can connect it with `any` ports, say with the `Print` component.
2. Compile the workflow. Confirm it compiles without error. Verify that in the compiled script, the `any` port properly set as an `InArg[any]`.
3. Use the workflow as a workflow component. Confirm that you can arbitrary data to it by compiling and running it. Confirm that the output is as expected.

## Tested on?
